app/containers/Header/index.js

```diff
@@ -149,21 +154,45 @@ const SearchWrap = styled(Box)`
   }
 `;
 
-const renderPages = ({ match, onClick, align }) =>
+const navButtonOnClick = ({ match, onClick, align, locale }) =>
   PAGES &&
   Object.values(PAGES)
     .filter(page => page.primary)
-    .map(page => (
-      <ButtonNavPrimary
-        key={page.key}
-        active={page.key === match}
-        disabled={page.key === match}
-        align={align}
-        onClick={() => onClick(page.key)}
-      >
-        <FormattedMessage {...rootMessages.page[page.key]} />
-      </ButtonNavPrimary>
-    ));
+    .map(page =>
+      page.key === 'download' ? (
+        <a
+          href={XPATHS.download[locale] || XPATHS.download[DEFAULT_LOCALE]}
+          target="_blank"
+          rel="noopener noreferrer"
+          key={page.key}
+        >
+          <ButtonNavPrimary>
+            <FormattedMessage {...rootMessages.page[page.key]} />
+            {/* prettier-ignore */}
+            <Share
+              color={
+                window.innerWidth > 991
```

Comment:
better use the theme breakpoints, also I am not quite sure what this is trying to achieve

also, icon could also be turned into a "styled icon", ie

```
const StyledShare = styled(Share)`
  color: ...
`
```

also

```
const StyledShare = styled(props => <Share {...props} size="small" />)`
  color: ...
`
```